alf/examples/ac_atari.gin

```diff
@@ -1,15 +1,16 @@
 # You need to install following packages
-# pip3 install atari-py opencv-python 
+# pip3 install atari-py opencv-python
 
 # environment config
 import tf_agents.environments.suite_atari
+import tf.environments.wrappers
```

Comment:
wrong name. Perhaps we should import alf.environments.wrapper in the python code, because it's used so often.

Reply:
moved the import to actor_critic.py